modules/references/client/components/CreateExperience.component.js

```diff
@@ -169,15 +191,13 @@ export default function CreateExperience({ userFrom, userTo }) {
         onSelect={() => {}}
         id="create-reference-tabs"
       >
-        <Tab eventKey={0} title={t('How do you know them')} disabled>
-          {tabs[0]}
-        </Tab>
-        <Tab eventKey={1} title={t('Recommendation')} disabled>
-          {tabs[1]}
-        </Tab>
-        <Tab eventKey={2} title={t('Feedback')} disabled>
-          {tabs[2]}
-        </Tab>
+        {tabs.map(({ component, title }, i) => {
+          return (
+            <Tab eventKey={i} key={`${i}`} title={title} disabled>
```

Comment:
How come not just `key={i}`? 

Index isn't perfect since list/index can change based on `sharedOnTime`? https://reactjs.org/docs/lists-and-keys.html

Reply:
> How come not just `key={i}`?

You know that I like overcomplicating things :D

> Index isn't perfect key here since list/index can change based on `sharedOnTime`? https://reactjs.org/docs/lists-and-keys.html

Fixed that, thanks!